components/Components/ChainTypeToggle.tsx

```diff
@@ -66,35 +83,46 @@ const ChainTypeToggle: React.FunctionComponent<ChainTypeToggleProps> = ({
         disabled={disabled}
         testID="settings.custom-server-chain.testnet"
         style={{ marginHorizontal: 5 }}
-        onPress={() => onPress(ChainNameEnum.testChainName)}>
+        onPress={() => onPress(ChainNameEnum.regtestChainName)}
```

Comment:
I don't really understand this change here, probably I'm missing something... (We are not using this component at all, and in the line 134 is the regtest toggle...)